src/fidesctl/cli/utils.py

```diff
@@ -185,7 +186,8 @@ def with_analytics(ctx: click.Context, command_handler: Callable, **kwargs: Dict
             except AnalyticsError:
                 pass  # cli analytics should fail silently
 
-def with_analytics_decorator(ctx: click.Context, command_handler: Callable, **kwargs: Dict) -> Any:  # type: ignore
+
+def with_analytics_decorator(func) -> Any:  # type: ignore
```

Comment:
I think return type could be `Callable` here?

Reply:
yeah true, will update the function doc also.